runtime/src/static/cpp/Config.cpp

```diff
@@ -72,19 +72,22 @@ void initializeConfig(int numThreads) {
   }
 }
 
-void initializeGlobal(int numThreads, size_t heapSize) {
+void initializeGlobal(int lowestCppTid, int numCppThreads, size_t heapSize, bool enablePCM) {
```

Comment:
enablePCM is required in both the framework and runtime for the following reasons:
(i) framework - to insert PCM instrumentation code into the MultiLoops if required
(ii) runtime - some of the instances of the flag such as the one you noted in the diff are remnants from old code. I've removed them. However, the Config object still needs to have an enablePCM field to decide whether to set the corresponding DDELITE_ENABLE_PCM flag in the Makefile